doozerlib/image.py

```diff
@@ -1077,6 +973,6 @@ async def find_non_latest_rpms(self) -> Dict[str, List[Tuple[str, str, str]]]:
         for arch in arches:
             iar = self.get_image_archive_inspector(arch)
             assert iar is not None
```

Comment:
We might use an error message in case the assertion fails that explains what is causing it and how it could be addressed